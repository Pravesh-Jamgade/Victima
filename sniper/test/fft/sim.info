{'cmdline': ['../../run-sniper',
             '-v',
             '-n',
             '1',
             '-c',
             '../../config/virtual_memory_configs/victima.cfg',
             '--roi',
             '--',
             './fft',
             '-p',
             '1'],
 'git_revision': '',
 'host': 'starship',
 'pin_version': 'unknown',
 'rusage': (3.305568,
            1.059034,
            478484,
            0,
            0,
            0,
            117850,
            0,
            0,
            0,
            568,
            0,
            0,
            0,
            729954,
            52),
 'snipercmd': ['bash',
               '-c',
               '/media/pravesh/Storage/code/sims/Victima/sniper/lib/sniper -c /media/pravesh/Storage/code/sims/Victima/sniper/config/base.cfg --general/total_cores=1 --general/output_dir=/media/pravesh/Storage/code/sims/Victima/sniper/test/fft --config=/media/pravesh/Storage/code/sims/Victima/sniper/config/nehalem.cfg --config=/media/pravesh/Storage/code/sims/Victima/sniper/config/utopia_motivation_baseline.cfg --config=/media/pravesh/Storage/code/sims/Victima/sniper/config/nehalem.cfg --config=/media/pravesh/Storage/code/sims/Victima/sniper/config/utopia_motivation_baseline.cfg --config=/media/pravesh/Storage/code/sims/Victima/sniper/config/virtual_memory_configs/common_translation.cfg --config=/media/pravesh/Storage/code/sims/Victima/sniper/config/virtual_memory_configs/victima.cfg -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpDx7zGY/run_benchmarks'],
 't_elapsed': 6.1652491092681885,
 't_start': 1747639778.187455,
 'tracecmds': [['/media/pravesh/Storage/code/sims/Victima/sniper/record-trace',
                '-o',
                '/tmp/tmpDx7zGY/run_benchmarks',
                '-v',
                '--roi',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './fft',
                '-p',
                '1']],
 'user': 'pravesh',
 'vmem': 22700032L}
