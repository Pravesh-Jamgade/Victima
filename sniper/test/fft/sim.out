                                   | Core 0    
  Instructions                     |    1133099
  Cycles                           |    2945596
  IPC                              |       0.38
  Time (ns)                        |    1132922
  Idle time (ns)                   |     216298
  Idle time (%)                    |      19.1%
Branch predictor stats             |           
  num correct                      |      30425
  num incorrect                    |       1679
  misprediction rate               |      5.23%
  mpki                             |       1.48
TLB Summary                        |           
  I-TLB                            |           
    num accesses                   |     100609
    num misses                     |          1
    miss rate                      |      0.00%
    mpki                           |       0.00
  D-TLB                            |           
    num accesses                   |     660638
    num misses                     |         10
    miss rate                      |      0.00%
    mpki                           |       0.01
  L2 TLB                           |           
    num accesses                   |         11
    num misses                     |         11
    miss rate                      |    100.00%
    mpki                           |       0.01
Cache Summary                      |           
  NUCA cache                       |           
    num cache accesses             |     235414
    num cache misses               |         11
    miss rate                      |      0.00%
    mpki                           |       0.01
DRAM summary                       |           
  num dram accesses                |        929
  average dram access latency (ns) |       0.00
  average dram queueing delay      |       0.00
Coherency Traffic                  |           
  num loads from dram              |        157
  num loads from dram cache        |          0
  num loads from remote cache      |          0
